hibernate-core/src/main/java/org/hibernate/MultiFindOption.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate;
+
+
+import jakarta.persistence.FindOption;
+
+/**
+ * Simple marker interface for FindOptions which can be applied to multiple id loading.
+ */
+public interface MultiFindOption extends FindOption {
```

Comment:
One more thing, since `MultiFindOption`:

-  doesn't actually add any type safety (it extends `MultiFindOption`)
- has a quite ugly name
- can be added into the hierarchy at any time if needed

I would just remove it, and make these new options implement `FindOption` directly.

It's not adding anything.

Reply:
It was steve's suggestion to do it like this.
It does make the restriction in `session.find(..., FindOption ... options)` somewhat cleaner having it around

Reply:
> It does make the restriction in `session.find(..., FindOption ... options)` somewhat cleaner having it around

I don't know what you mean by this.

Reply:
> It's not adding anything.

I disagree.  Find usages of MultiFindOption has value.

> has a quite ugly name

Your opinion.

Reply:
> is quite nicer than individually checking for each type.

Which is what I meant